code/modules/mob/living/carbon/update_icons.dm

```diff
@@ -193,6 +193,9 @@
 
 /mob/living/carbon/update_body()
 	update_body_parts()
+	// snowflakey thing: update blood colour if a slimeperson
+	if(isjellyperson(src))
+		dna.species.exotic_blood_color = "#" + dna.features["mcolor"]
```

Comment:
Don't you have anywhere else (like species/proc/handle_mutant_bodyparts) to move this string of code to?

Reply:
ill try moving it into there, didn't think about that

Reply:
placed it in the handle_mutant_bodyparts method for jellypeople